nextjs/src/app/(app)/reports/ReportList.tsx

```diff
@@ -100,25 +101,26 @@ export function ReportCard({
 }) {
   return (
     <Link href={`/reports/${report.id}`}>
-      <Card>
-        <CardHeader>
-          <CardContent>
-            <Image
+      <Card className="hover:bg-meepGray-700 transition-colors duration-200">
+        <CardHeader className=" p-4">
+          <CardContent className="">
+            <Map className="w-4 text-brandBlue" />
+            {/* <Image
               src="/reports_page_card_image.png"
               alt="Description of the image"
               width={300}
               height={300}
               className="w-auto"
-            />
+            /> */}
```

Comment:
suggestion: remove this commented-out code, if it's not being used

Reply:
Correct this and some other stuff i missed.